### PR TITLE
DOC: Minor doc fixes

### LINKED ISSUE #383: DOC: Raising issues
Some of the docs don't specify that issues are being raised

### PR DESCRIPTION
**Pull request by ElDeveloper**
@mortonjt or @antgonza, should be really small, only doc changes.

Fix #383

### CODE REVIEW

**Comment by emperor-helper** [comment]
The test build for this pull request can be found here: http://emperor.colorado.edu/pull_390/make_emperor/

**Comment by mortonjt** [comment]
Looks good to me!

File: emperor/biplots.py [hunk 1]
```diff
@@ -100,6 +100,11 @@ def preprocess_otu_table(otu_sample_ids, otu_table, lineages,
     lines : np.ndarray
          coords where the N most prevalent taxa will be positioned in the
          biplot
+
+    Raises
+    ------
+    EmperorUnsupportedComputation
+        If one or fewer rows are passed in the contingency table.
```
Comment (mortonjt) [inline emperor/biplots.py]: Maybe reword to "If the contingency table contains one or fewer rows"?
